src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py

```diff
@@ -12352,6 +12382,18 @@ def _quantiles_single_col(
 
         return SnowflakeQueryCompiler(internal_frame)
 
+    def mode(
+        self, axis: Axis = 0, numeric_only: bool = False, dropna: bool = True
```

Comment:
can those method not implemented error be at the overwritten part? instead of query compiler like how other apis works today

Reply:
API not implemented errors today are all thrown at frontend api layer, so let's keep it the same everywhere, if we decide to do those not implemented error at query_compiler layer, let's do it the same for all API also